components/SwipeForJobs/SwipeForJobs.tsx

```diff
@@ -4,25 +4,45 @@ import { View } from "react-native";
 import { JobCard } from "../JobCard/JobCard";
 import { styles, overlayLabels } from "./styles";
 import { Job } from "../Job";
+import AsyncStorage from "@react-native-async-storage/async-storage";
 
 type SwipeForJobsProps = {
   jobs: Job[];
 };
 
 export const SwipeForJobs = ({ jobs }: SwipeForJobsProps) => {
   const [index, setIndex] = React.useState<number>(0);
-
   const onSwiped = (): void => {
     setIndex((index + 1) % jobs.length);
   };
 
+  const saveOrRejectCurrentJob = (key: string, index: number) => {
+    AsyncStorage.getItem(key, (err, result) => {
```

Comment:
looks like the return type for getItem() is a Promise. typically, you want to resolve a promise using .then or your favorite, async await. here's an example on https://stackoverflow.com/questions/37120305/waiting-for-asyncstorage-getitem. also you can add error handling if the promise fails